### PR TITLE
Allow using a newer version of protobuf

### PR DESCRIPTION
**Pull request by nehalkpatel**
Will require recompiling protos

- - -
This change is [Reviewable](https://reviewable.io/reviews/google/openhtf/1040)

### CODE REVIEW

**Comment by nehalkpatel** [comment]
I'll leave it up to the maintainers if they want to merge this, since using a newer protoc _may_ introduce breaking changes?

**Comment by nbai2022** [comment]
This is for external setup, LGTM

**Review by nbai2022 — APPROVED** [review]

**Pull request merged**
